### DISCUSSION TITLE
Rename repository

### DISCUSSION THREAD

**Discussion by SuspiciousLookingOwl** [SuspiciousLookingOwl/web-frameworks-benchmark#5] [General]
Should we rename this repository? Perhaps we can delete [the-benchmarker/website](https://github.com/the-benchmarker/website) repository and rename this repository to `website`. Also what's gonna happen to [the-benchmarker/frontend](https://github.com/the-benchmarker/frontend)?

/cc @waghanza

**Comment by waghanza** [comment]
Sure, website seems more relevant for relevant for me (since my big plan is to have 3 kinds of benchmarks : full stack / backend / frontend)

- full-stack, probably though https://github.com/gothinkster/realworld
- backend, from the current project
- frontend, to be defined ...

what do you think @SuspiciousLookingOwl ?

PS : Maybe archive frontend, and warn about new repo location in `README`

**Reply by SuspiciousLookingOwl** [reply]
I agree with using `website` as the repository name.

> Maybe archive frontend, and warn about new repo location in README

Sounds good!

**Comment by SuspiciousLookingOwl** [comment]
Hi @waghanza, can I change the repo name now?

**Reply by waghanza** [reply]
sure, I think you have the rights to do so ...
